lms/templates/fields/field_order_history.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="field u-field-order">
+    <span class="u-field-order-title"><%- title %></span>
+    <span class="u-field-order-date"><%- orderDate %></span>
+    <span class="u-field-order-value u-field-order-price"><% if (!isNaN(parseFloat(totalPrice))) { %>$<% } %><%- totalPrice %></span>
+    <span class="u-field-order-value u-field-order-number"><%- orderId %></span>
+    <span class="u-field-order-link">
+        <% if (receiptUrl) { %>
+            <a class="u-field-link hidden" aria-live="polite" target="_blank" href="<%- receiptUrl %>"><%- gettext('Order Details') %></a>
```

Comment:
I was not able to tab navigate to this link. It appears to be because you are using the hidden class. Please update to toggle opacity or some other property that will enable tab navigation to the links. What is aria-live used for in this context? It would be good to add screen reader context to the link text (otherwise it is a list of links that all say 'Order Details' without stating this order the details are for)